### ISSUE TITLE
GitHub Branches - best practice 

### ISSUE THREAD

**Issue by jpradocueva** [OpenManufacturingPlatform/rules_of_engagement#7]
This is a **proposal** for GitHub branches in OMP, based on Friday 15th discussion:
(it differs slightly from current [diagram](https://github.com/OpenManufacturingPlatform/rules_of_engagement#github-flow) )

* `feature-branch`:
   * It is created from the `development` branch by the WG chair
   * A `feature-branch' is really anything you want it to be—a bug fix, new functionality, or even just more documentation
   * Feature-branches should be merged regularly into the `development` branch.
   * Pull Requests against these branches are agreed by the WG and merged by the WG chair
>Note:
>  It is used to implement a single feature in the specification. If two different `features-branches` update the same statements in a document then, a conflicts will occur.

* `development` branch:
   * It is created from the `master` branch by the Maintainer.
   * When a feature is complete it is merged into the `development` branch by the WG Chair.
   * A`development` branch that is formally agreed MUST NOT contain incomplete features.

* `master` branch:
   * It is used as a publication branch.
   * It contains documents that have been agreed formally agreed by the Working Group and ratified by the Steering Committee.
   * A `development` branch will be merged into `master`  branch only after the Steering Committee has ratified the content in the `development` branch
   * Only maintainers SHOULD merge `development` branches into.

* `pre-publication` or `staging` branch:
   * This branch MAY be used in some cases as an intermediate step between the `development` and `master` branch. 
   * It is used as a pre-publication branch in case there is the need for continuing developing the specifications while the final content is awaiting Working Group or Steering Committee approval.

Upon discussion and agreement, I will create a Pull Request

**Comment by jpradocueva** [comment]
incorporated in  the GitHub Flows diagram
